torch/_torch_docs.py

```diff
@@ -12419,6 +12419,9 @@ def merge_dicts(*dicts):
     y (Tensor or Scalar): value (if :attr:`y` is a scalar) or values selected at indices
                           where :attr:`condition` is ``False``
 
+Keyword args:
+    {out}
```

Comment:
Is this rendered correctly? I think it will not, as this function is not passed the relevant dictionary to make hte substitution.

Reply:
Thank you for pointing that out. Updated the PR accordingly :)